urwid/tests/test_signals.py

```diff
@@ -1,19 +1,7 @@
 import unittest
+from unittest.mock import Mock
 
-try:
```

Comment:
re-sort imports (automatic change)

we're on python3, so try-except is not needed